projectile.el

```diff
@@ -502,7 +507,35 @@ just return nil."
         version)
     (error "Cannot determine version without package pkg-info")))
 
-
+;;; Workarounds
+
+(eval-after-load 'etags
```

Comment:
ok, I'll try to update this PR tonight / tomorrow. Ping me if I forget (it would be very useful if github allowed one to mark a PR / ticket as "unread")